data/discord/messages/homeassistant.yaml

```diff
@@ -1388,3 +1388,7 @@ safe_mode:
 
     **NOTE:** Safe mode comes in Home Assistant Core 2023.11.0 or later.  
     
+armbian:
+  description: "Where to get help with Supervised on Armbian"
+  content: >-
+    We can't help you here with the Armbian Supervised installer. Instead please use [their forum](https://forum.armbian.com/topic/33245-armbian-with-preinstalled-home-assistant-supervised/).
```

Comment:
Should we be clear on the fact this isn't created or supported by the Home Assistant project?